### ISSUE TITLE
řČí looks ugly

### ISSUE THREAD

**Issue by certik** [ipython/xkcd-font#29]
If I type my name (Ondřej Čertík) into the test app, it looks really ugly:

https://cdn.rawgit.com/ipython/xkcd-font/master/preview.html?text=Ond%C5%99ej%20%C4%8Cert%C3%ADk

**Comment by takluyver** [comment]
I don't think we have glyphs for any accented characters, so it probably falls back to another font for those.

**Comment by certik** [comment]
Yes, it's due to #11, so I commented there what would be needed to fix this issue: https://github.com/ipython/xkcd-font/issues/11#issuecomment-326470048

**Comment by pelson** [comment]
Děkujeme za nahlášení, Ondřeji Čertíku, trvalo to sice déle, ale nyní bychom už měli mít všechny glyfy, které potřebujete. (Příliš žluťoučký kůň úpěl ďábelské ódy, šťastně běží přes čtvrť s řemeslníky.)

(translated) Thank you for reporting this, @certik. It took a while, but we should now have all the glyphs you need. (An overly yellowish horse groaned devilish odes, happily running through a district with craftsmen.)


[<img width="1046" height="192" alt="Image" src="https://github.com/user-attachments/assets/4e0e8296-532d-4a61-b0be-3502db7e91bb" />Live demo](https://ipython.org/xkcd-font/preview.html?text=D%C4%9Bkujeme%20za%20nahl%C3%A1%C5%A1en%C3%AD%2C%20Ond%C5%99eji%20%C4%8Cert%C3%ADku%2C%20trvalo%20to%20sice%20d%C3%A9le%2C%20ale%20nyn%C3%AD%0Abychom%20u%C5%BE%20m%C4%9Bli%20m%C3%ADt%20v%C5%A1echny%20glyfy%2C%20kter%C3%A9%20pot%C5%99ebujete.%0A%0A(P%C5%99%C3%ADli%C5%A1%20%C5%BElu%C5%A5ou%C4%8Dk%C3%BD%20k%C5%AF%C5%88%20%C3%BAp%C4%9Bl%20%C4%8F%C3%A1belsk%C3%A9%20%C3%B3dy%2C%20%C5%A1%C5%A5astn%C4%9B%20b%C4%9B%C5%BE%C3%AD%20p%C5%99es%20%C4%8Dtvr%C5%A5%20s%20%C5%99emesln%C3%ADky.)%0A%0A%0A)


Feedback on the rendering welcome - we can tweak quite a lot of parameters still...

**Comment by certik** [comment]
@pelson beautiful! 

My only feedback is for `ť`, such as in `žluťoučký`, make the space after it smaller, I think either completely or almost equivalent to just regular `t`, and then put the `'` accent on it, but don't change the kerning, or just tiny bit. Right now it looks almost like `žlut'oučký` instead of `žluťoučký`. 

Other than that it looks great!

**Comment by pelson** [comment]
Proposal in https://github.com/ipython/xkcd-font/pull/59#issue-4281201126 should close this out. Happy to take feedback still (even after merge).

**Comment by certik** [comment]
I commented there, it's fixed!

**Comment by pelson** [comment]
Thanks for the feedback! The [preview](https://ipython.org/xkcd-font/preview.html?text=D%C4%9Bkujeme%20za%20nahl%C3%A1%C5%A1en%C3%AD%2C%20Ond%C5%99eji%20%C4%8Cert%C3%ADku%2C%20trvalo%20to%20sice%20d%C3%A9le%2C%20ale%20nyn%C3%AD%0Abychom%20u%C5%BE%20m%C4%9Bli%20m%C3%ADt%20v%C5%A1echny%20glyfy%2C%20kter%C3%A9%20pot%C5%99ebujete.%0A%0A(P%C5%99%C3%ADli%C5%A1%20%C5%BElu%C5%A5ou%C4%8Dk%C3%BD%20k%C5%AF%C5%88%20%C3%BAp%C4%9Bl%20%C4%8F%C3%A1belsk%C3%A9%20%C3%B3dy%2C%20%C5%A1%C5%A5astn%C4%9B%20b%C4%9B%C5%BE%C3%AD%20p%C5%99es%20%C4%8Dtvr%C5%A5%20s%20%C5%99emesln%C3%ADky.)%0A%0A%0A) should be now updated after #59 got merged.

**Comment by certik** [comment]
@pelson, thank you so much for fixing it. It really works well now.
